qcodes/instrument/group_parameter.py

```diff
@@ -1,3 +1,10 @@
+"""
+This module implements a :class:`Group` intended to hold multiple
+parameters that are to be get and set by the same command. The parameters
```

Comment:
I guess we use "gotten" instead of "get" here? I know that it's a controversial word, but I think that it is what we normally use.

Reply:
"gotten" and "setten" ? :)

I'd actually agree with that especially because it allows to distinguish between "get" method and "get" word.

Reply:
Setten? -Get outta here!